contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -103,65 +103,34 @@ contract RyskStrategy is BaseStrategy {
     }
 
     /// @inheritdoc IStrategy
-    /// @notice this also includes shares from the pending withdrawal and shares yet to be minted for pending deposits
     function investedAssets()
         public
         view
         virtual
         override(IStrategy)
         returns (uint256)
     {
-        IRyskLiquidityPool.DepositReceipt
-            memory depositReceipt = _getDepositReceipt();
-        IRyskLiquidityPool.WithdrawalReceipt
-            memory withdrawalReceipt = _getWithdrawalReceipt();
+        uint256 amountInDepositReceipt = _getAmountFromDepositReceipt();
+
         uint256 currentWithdrawalEpoch = ryskLqPool.withdrawalEpoch();
         // since withdrawal price per share is not updated until the end of the epoch,
         // we need to use the price per share from the previous epoch
         uint256 currentWithdrawalPricePerShare = ryskLqPool
-            .withdrawalEpochPricePerShare(ryskLqPool.withdrawalEpoch() - 1);
-
-        // shares for pending deposit (not yet minted or not yet updated on the deposit receipt as unredeemed)
-        uint256 amountInPendingDepositShares = _getAmountInSharesForPendingDeposit(
-                depositReceipt,
-                currentWithdrawalPricePerShare
-            );
+            .withdrawalEpochPricePerShare(currentWithdrawalEpoch - 1);
 
-        // shares marked for withdrawal that are now owned by the pool itself
-        uint256 amountInPendingWithdrawalShares = 0;
-        if (withdrawalReceipt.epoch != 0) {
-            if (withdrawalReceipt.epoch == currentWithdrawalEpoch) {
-                amountInPendingWithdrawalShares = _sharesToUnderlying(
-                    withdrawalReceipt.shares,
-                    currentWithdrawalPricePerShare
-                );
-            } else {
-                amountInPendingWithdrawalShares = _sharesToUnderlying(
-                    withdrawalReceipt.shares,
-                    ryskLqPool.withdrawalEpochPricePerShare(
-                        withdrawalReceipt.epoch
-                    )
-                );
-            }
-        }
-
-        // unredeemed shares are not counted in the strategy's balance
-        // because they are not yet claimed (they are still owned by the pool)
-        uint256 amountInUnredeemedShares = _sharesToUnderlying(
-            depositReceipt.unredeemedShares,
+        uint256 amountInPendingWithdrawal = __getAmountPendingWithdrawal(
```

Comment:
why the double underscore?

Reply:
my mistake, [fixed](https://github.com/lindy-labs/sc_solidity-contracts/pull/216/commits/070e898758ce4800b05c9f4acdd08fa37cd1ebd2)